### DISCUSSION TITLE
How do I reduce the bottom margin size for the page?

### DISCUSSION THREAD

**Discussion by tabreturn** [Submanifold/latex-mimosis#53] [Q&A] [answered]
I want to make the bottom margin a bit narrower. How would I achieve this? Thank in advance.

**Comment by Pseudomanifold** [comment, marked as answer]
The easiest thing would be to use the `geometry` package:

- Add `usegeometry` to the main options of the class
- Load `geometry` with the `bottom` option, for instance: `\usepackage[bottom = 1cm]{geometry}`

**Reply by tabreturn** [reply]
Thanks for the quick response! I've tried this already but have yet to succeed. I added the line to the main `.tex` file, and also tried adding it to the `.cls` file.

Do you have an example I could refer to?

**Reply by Pseudomanifold** [reply]
What's the error that you are getting?

**Reply by tabreturn** [reply]
Just spotted the problem! 

I had a `\newgeometry` on my title page, which was overwriting the change. Thank you for your help and a great template!

**Reply by Pseudomanifold** [reply]
Great, thanks a lot!
